scripts/mixins/families/animated_weapons.lua

```diff
@@ -35,7 +35,7 @@ g_mixins.families.animated_weapons = function(animatedMob)
         mob:SetMagicCastingEnabled(true)
         mob:SetAutoAttackEnabled(true)
         mob:SetMobAbilityEnabled(true)
-        mob:setLocalVar("Text", dialogChoice[mob:getName()])
+        mob:setLocalVar("Text", dialogChoice[mob:getName()]) -- Fix ME
```

Comment:
What does the `Fix me` mean here?

Reply:
This was already fixed, it was dealing with the name lookups for their text. This was the reason for the huge renaming piece I did related to animated weapons.